src/home/rendering/display-github-user-information.ts

```diff
@@ -37,12 +37,14 @@ export async function displayGitHubUserInformation(gitHubUser: GitHubUser) {
     window.location.reload();
   });
 
+  containerDivElement.appendChild(authenticatedDivElement);
```

Comment:
Is this correct to append? I expected only changes to tests.

Reply:
@nick-dev123 Why was the UI updated?